### ISSUE TITLE
language choice in @code

### ISSUE THREAD

**Issue by vinniefalco** [cppalliance/mrdox#244]
Apparently this works in doxygen? `@code{cpp}`?

**Comment by klemens-morgenstern** [comment]
Yes, I looked into our code and can't figure out a solution short of parsing the content of `.code`.

That could of course work, e.g;

```cpp
// language=cpp
```

for C++

and 

```bash
# language=bash

```

for bash.

**Comment by vinniefalco** [comment]
Well we have to parse the content, obviously. Its kind of annoying. But I will do it.
